ui-tests/src/test/java/org/openmrs/reference/DeleteRequestAppointmentTest.java

```diff
@@ -41,8 +41,8 @@ public void deleteRequestAppointmentTest() {
         ClinicianFacingPatientDashboardPage patientDashboardPage = activeVisitsPage.goToPatientDashboardOfLastActiveVisit();
         RequestAppointmentPage requestAppointmentPage = patientDashboardPage.clickOnRequest();
         requestAppointmentPage.enterAppointmentType(SERVICE_NAME);
-        requestAppointmentPage.enterValue("0");
-        requestAppointmentPage.selectUnits("Day(s)");
+        requestAppointmentPage.enterMinimumValue("0");
+        requestAppointmentPage.selectMinimumUnits("Day(s)");
```

Comment:
Refer to the comment [above](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/491/commits/48b65dcdb8e155aa5fa9a090c6413580ef7650ae#r668604884)